tests/pages/Welcome.test.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import configureStore from 'redux-mock-store';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import axios from 'axios';
+import waitForExpect from 'wait-for-expect';
+import { StatusCodes } from 'http-status-codes';
+import Welcome from '../../src/pages/Welcome';
+
+const mockStore = configureStore([]);
+jest.mock('axios');
+
+function getComponent(store) {
+  return renderer.create(
+    <Provider store={store}>
+      <Welcome />
+    </Provider>
+  );
+}
+
+describe('Welcome', () => {
+  let store;
+
+  beforeEach(() => {
+    store = mockStore({
+      authenticated: false
+    });
+  });
+
+  it("renders login correctly", () => {
+    const tree = getComponent(store).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it("renders change password correctly", () => {
+    const tree = getComponent(mockStore({
+      authenticated: false,
```

Comment:
```suggestion
      authenticated: true,
```

The snapshot currently doesn't render change password. Make sure to verify the generated snapshots when creating snapshot tests.